tests/test_crew.py

```diff
@@ -1,19 +1,19 @@
 from datadog import statsd
 import logging
+import mock
```

Comment:
This import is unused; it should be removed.
It should also otherwise be `from unittest import mock`

Reply:
Thanks for the catch. I am getting rid of this